recipes-core/systemd/systemd.inc

```diff
@@ -6,16 +6,22 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
 SRC_URI += " \
     file://20-network-io.conf \
+    file://10-any_interface_is_enough.conf \
 "
 
 FILES:${PN} += "\
     ${sysconfdir}/sysctl.d/20-network-io.conf \
+    ${sysconfdir}/systemd/system/systemd-networkd-wait-online.service.d/10-any_interface_is_enough.conf \
```

Comment:
why `${sysconfdir}/systemd/system`  and not `${systemd_system_unitdir}`? The latter seems to be the more appropriate location.

Reply:
Done.

Reply:
You should update this line as well, as these are two different locations ({sysconfdir} is /etc, while {systemd_system_unitdir} is /lib/..)

Reply:
Right. Missed that.